packages/cli/src/license/license.controller.ts

```diff
@@ -1,129 +1,37 @@
-import express from 'express';
-import { Container } from 'typedi';
-
-import { Logger } from '@/Logger';
-import * as ResponseHelper from '@/ResponseHelper';
-import type { ILicensePostResponse, ILicenseReadResponse } from '@/Interfaces';
-import { LicenseService } from './License.service';
-import { License } from '@/License';
-import type { AuthenticatedRequest, LicenseRequest } from '@/requests';
-import { InternalHooks } from '@/InternalHooks';
-import { UnauthorizedError } from '@/errors/response-errors/unauthorized.error';
-import { BadRequestError } from '@/errors/response-errors/bad-request.error';
-
-export const licenseController = express.Router();
-
-const OWNER_ROUTES = ['/activate', '/renew'];
-
-/**
- * Owner checking
- */
-licenseController.use((req: AuthenticatedRequest, res, next) => {
-	if (OWNER_ROUTES.includes(req.path) && req.user) {
-		if (!req.user.isOwner) {
-			Container.get(Logger).info('Non-owner attempted to activate or renew a license', {
-				userId: req.user.id,
-			});
-			ResponseHelper.sendErrorResponse(
-				res,
-				new UnauthorizedError('Only an instance owner may activate or renew a license'),
-			);
-			return;
-		}
+import { Service } from 'typedi';
+import { Authorized, Get, Post, RequireGlobalScope, RestController } from '@/decorators';
+import { LicenseRequest } from '@/requests';
+import { LicenseService } from './license.service';
+
+@Service()
+@Authorized()
+@RestController('/license')
+export class LicenseController {
+	constructor(private readonly licenseService: LicenseService) {}
+
+	@Get('/')
+	async getLicenseData() {
```

Comment:
Should we start using `@NoAuthRequired()` to make this more explicit?

Reply:
but, so far auth has been required for this endpoint. that's why there is an `@Authorized` on the class.

Reply:
Ah of course!